### DISCUSSION TITLE
Django cannot find module netbox_topology_views

### DISCUSSION THREAD

**Discussion by fribse** [mattieserver/netbox-topology-views#461] [General]
### NetBox version

v3.7.3

### Topology Views version

v3.9.0

### Steps to Reproduce
Python 3.10.12
pip 24.0

When tried to install outside venv
Steps with manage.py migrate failed, django reports module not found
uninstalled the netbox-topology-views (pip3 uninstall)
started venv
installed again
django still fails on module not found
 

### Expected Behavior

When uninstallation is performed, I expect it to install properly afterwards.

### Observed Behavior

It looks like some requirements are not fullfilled.

**Comment by dreng** [comment]
That does not look like a bug to me. Unfortunately, you've made a mistake when you followed the procedure. Who knows what has been broken when you did that. It is completely out of scope of the plugin to repair any user mistakes.

I'm converting this question into a discussion.

**Reply by mariano-daniel** [reply]
I would recommend that the README / instructions is updated with more clear steps to follow.  The Docker notes should be more visually separated from the instructions and make also the instructions more clear. 
There's a few steps at the beginning and then there's a step in the middle that's worded "First run". Should I do that step first or follow from top to bottom?

If you could use a numbered bullet point list that would help us a lot!
